lib/xandra/protocol.ex

```diff
@@ -378,7 +394,7 @@ defmodule Xandra.Protocol do
   end
 
   defp encode_value({:udt, fields}, value) when is_map(value) do
-    for {field_name, field_type} <- fields do
+    for {field_name, [field_type]} <- fields do
```

Comment:
Can you clarify this one?

Reply:
The before-shape conflicts with all other type definitions—it is always {type, [type]}.
Before it does not really matter, but now we need to unify it for easier usage in `enhance_column_types/2`.